third_party/move/move-compiler-v2/tests/checking/typing/implicit_deref_borrow_field_from_non_struct.exp

```diff
@@ -1,49 +1,49 @@
 
 Diagnostics:
-error: type `integer` cannot be resolved as a struct
+error: constraint `<some struct>{f: ?0}` incompatible with `integer`
```

Comment:
Yes, in general this message has been improved now. However, the "expected" isn't working because the polarity is unknown where this error is produced. But I think it is good enough for now.